src/protocols/MeterSML.cpp

```diff
@@ -244,6 +250,7 @@ ssize_t MeterSML::read(std::vector<Reading> &rds, size_t n) {
 	sml_file *file;
 	sml_get_list_response *body;
 	sml_list *entry;
+	bool direction_from_status = false;
```

Comment:
this variable has the same name as the property that stores the setting,
which might be confusing.